interfaces/automated-testing/scd/scd_injection_observation.yaml

```diff
@@ -0,0 +1,792 @@
+            Contingent state, then this field must be populated.  When this
+            field is populated, an implicit subscription will be created and
+            associated with this operational intent, and will generally be
+            deleted automatically upon the deletion of this operational intent.
+          anyOf:
+          - $ref: '#/components/schemas/ImplicitSubscriptionParameters'
+
+    OperationalIntentList:
+      description: A list of operational intents
+      type: array
+      items: 
+        $ref: '#/components/schemas/OperationalIntent'
+    Key:
+      description: >-
+        Proof that a client has obtained the latest airspace content, used to
+        ensure that write operations to the DSS occur only when the latest
+        content is known (i.e. has been read). The client is required to pass a
+        current Key constructed from information obtained with previous read
+        operations and subsequent requests for full information, and optionally,
+        subscription updates for deconflicted write operations to the DSS.  The
+        contents of this data structure are generated by the client.
+      type: array
+      items:
+        $ref: '#/components/schemas/EntityOVN'
+      default: [ ]
+    DeleteTestResponse:
+      description: ''
+      required:
+        - operational_intents
+      type: object
+      properties:
+        operational_intents:
+          description: 'The complete set of test data deleted.'
+          type: array
+          items:
+            $ref: '#/components/schemas/OperationalIntent'
+
+
+    CreateTestParameters:
+      description: Collection of fully-defined operational intents
+      type: object
+      properties:
+        test_id: 
+          description: ID provided when a test payload is submitted to the system under test
+          type: string
+          format: uuid 
+        operational_intents:
+          description: A list of operational intents provided for the test
+          $ref: '#/components/schemas/OperationalIntentList'
+
+    FlightNotAuthorizedResult:  
+      type: object
+      properties:
+        scd_processing_result:  
+          type: string        
+          enum: [OperationalConflict, 'CouldNotProcess']   
+          description: >
+              Sort order:
+                * `OperationalConflict` - The Flight not Authorized becuase of a conflict with another operation
+                * `CouldNotProcesss` - The OperationalIntent could not be processed by the USSP, this might be because of a number of reasons e.g. the intent is so far in the future the USSP cannot process it.
+        area_of_interest:
+          type: object
+          $ref: '#/components/schemas/Volume4D'
+                  
+      
+    SCDProcessingResults: 
+      description: Results for the operational intents submitted to the create test endpoint
+      type: array
+      items:
+        anyOf:
```

Comment:
We should not use polymorphism here because most code generation tools will not be able to render this data type.  Instead, there should be two separate fields (perhaps `success_result` and `not_authorized_result` of types `OperationalIntent` and `FlightNotAuthorizedResult`, respectively) where exactly one is populated, per the description.

Reply:
I have now created two objects showing the success or failure response.